service/provider/src/main/java/gov/va/vro/service/provider/camel/MasIntegrationRoutes.java

```diff
@@ -112,10 +112,12 @@ public class MasIntegrationRoutes extends RouteBuilder {
   private final HealthAssessmentErrCheckProcessor healthAssessmentErrCheckProcessor;
 
   // Possible OffRamp Reasons
-  public static final String SUFFICIENCY_UNDETERMINED = "Sufficiency cannot be determined.";
+  public static final String SUFFICIENCY_UNDETERMINED =
+      "Sufficiency cannot be determined. Reason: assessorError.";
   public static final String PDF_UPLOAD_FAILED_AFTER_ORDER_EXAM = "docUploadFailed";
   public static final String EXAM_ORDER_FAILED = "examOrderFailed";
-  public static final String NEW_NOT_PRESUMPTIVE = "newClaimMissingFlash266";
+  public static final String NEW_NOT_PRESUMPTIVE =
+      "New claim cannot be determined to be presumptive. Reason: newClaimMissingFlash266.";
   public static final String ANNOTATIONS_FAILED = "annotationDataRequestFailed";
```

Comment:
Please leave these as keys and change SUFFICIENCY_UNDETERMINED to be a key to. I believe others are using these to make decisions. At the place where the actual slack messages are generated, add the narrative based on key.

Reply:
Changed the code to use an object EventReason for reason code and narrative. See comments in MCP-2679.